### PR TITLE
Replace `Award.relatedBid` with `Award.relatedBids`

### LINKED ISSUE #125: Bids: Make awards.relatedBid an array
In some cases the tenderers can make multiples bids, one for each lot or item that are being tender. And then, the procuring entity can make an award that includes differents items/lots that were bidded in differents bids. 

For example, the EU has "The contracting authority reserves the right to award contracts combining the following lots or groups of lots".

Due that, OCDS should have a way to combine multiple bids on different lots into one award.

To make that possible the suggestion is that the `Award.relatedBid` field change to `Award.relatedBids` and become an array instead of a string.

cc @jpmckinney

### PR DESCRIPTION
**Pull request by duncandewhurst**
Closes [#125](https://github.com/open-contracting/ocds-extensions/issues/125)

### CODE REVIEW

**Comment by duncandewhurst** [comment]
The checks are failing with the following error:

> test_readme.py::test_example_backticks
>  README.md: "Award.relatedBid" term is not in schema

However, `Award.relatedBid` is in the schema, it's just deprecated.

**Review by mrshll1001 — APPROVED** [review]
This looks good.

**Comment by jpmckinney** [comment]
We need to mark the old field as deprecated, since the governance process applies to this extension.

**Comment by duncandewhurst** [comment]
Ah looks like `ocdskit indent` unexpectedly removed the old property and deprecation note because I had accidentally duplicated `definitions/Award/properties. I've added it back in https://github.com/open-contracting-extensions/ocds_bid_extension/pull/46/commits/af83df67b212e0b8702a7ea38fd25ac3c76f24cf and opened https://github.com/open-contracting/ocdskit/issues/182 in case we want to change that behaviour.

File: release-schema.json [hunk 1]
```diff
@@ -15,7 +15,22 @@
           "type": [
             "string",
             "null"
-          ]
+          ],
+          "deprecated": {
+            "description": "This field is deprecated in favour of `.relatedBids`, to support combining multiple bids into one award.",
```
Comment (jpmckinney) [inline release-schema.json]:
```suggestion
            "description": "This field is deprecated in favor of `relatedBids`, to allow an award to combine multiple bids.",
```

File: README.md [hunk 2]
```diff
@@ -122,6 +124,7 @@ Report issues for this extension in the [ocds-extensions repository](https://git
 
 ### Unreleased
 
+* Replace `Award.relatedBid` with `Award.relatedBids`
```
Comment (jpmckinney) [inline README.md]:
```suggestion
* Add `Award.relatedBids` field, and deprecate `Award.relatedBid` field
```

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**